src/settings_gui.cpp

```diff
@@ -215,6 +235,14 @@ struct GameOptionsWindow : Window {
 				}
 				break;
 
+			case WID_GO_REFRESH_RATE_DROPDOWN: // Setup refresh rate dropdown
+				for (uint i = 0; i < _refresh_rates.size(); i++) {
+					auto item = new DropDownListParamStringItem(STR_GAME_OPTIONS_REFRESH_RATE_ITEM, i, false);
+					item->SetParam(0, *std::next(_refresh_rates.begin(), i));
```

Comment:
I feel like it's probably nicer to use an iterator in the for loop, rather than the (repeated) std::next stuff (then use std::distance / `it - foo.begin()` to get the index)

Think you might even be able to do a range-for loop, if you can manage to convince it to grab the iterator

Saves having an O(n^2) loop too :)

Alternatively, using sets makes all that impossible, in which case you should ignore the above